python/lsst/validate/drp/validate.py

```diff
@@ -202,6 +209,22 @@ def runOneRepo(repo, dataIds=None, metrics=None, outputPrefix='', verbose=False,
     The filter name is added to this prefix.  If the filter name has spaces,
     there will be annoyance and sadness as those spaces will appear in the filenames.
     """
+    from lsst.daf.persistence import Butler
+    if instrument is None:
+        instrument = extract_instrument_from_repo(butler)
+    if dataset_repo_url is None:
+        dataset_repo_url = repo
```

Comment:
I'm not sure the full path to the repo is very helpful.  It seems more useful to just truncate on the last part of the path, or to try to discover the actual repository name by asking `git` or by inspecting `.git/config` in the repo.  I think this would be easy with `gitpython`, but not sure if we want that dependence.

Reply:
We shoudn't assume that the data are being read from a repository that is a git repository.
  * People should be able to run this on data that aren't tracked in git.
  * In fact, packages *installed* by lsstsw don't actually have the .git included.

If the caller wants it to be based on the git information they can do the lookup and pass to the script.

In the absence of having specified something, the full path is more informative.

Reply:
Good enough.